src/__tests__/actionTests/profileActionTests.js

```diff
@@ -0,0 +1,164 @@
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 404,
+        response: {
+          message: 'Error',
+        },
+      });
+    });
+    const expectedActions = [{
+      payload: {
+        message: 'Error',
+      },
+      type: FETCH_PROFILE_ERROR,
+    }];
+    store = mockStore({});
+    await store.dispatch(getProfile())
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+
+  it('Should dispatch UPDATE_PROFILE_SUCCESS', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          message: 'Success',
+        },
+      });
+    });
+    const expectedActions = [{
+      payload: 'Success',
+      type: UPDATE_PROFILE_SUCCESS,
+    }];
+    store = mockStore({});
+    await store.dispatch(updateProfile())
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+
+  it('Should dispatch UPDATE_PROFILE_ERROR', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 404,
+        response: {
+          message: 'Error',
+        },
+      });
+    });
+    const expectedActions = [{
+      payload: {
+        message: 'Error',
+      },
+      type: UPDATE_PROFILE_ERROR,
+    }];
+    store = mockStore({});
+    await store.dispatch(updateProfile())
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+
+  it('Should dispatch UPLOAD_IMAGE_SUCCESS', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          secure_url: 'Success',
+        },
+      });
+    });
+    const expectedActions = [{
+      payload: 'Success',
+      type: UPLOAD_IMAGE_SUCCESS,
+    }];
+    store = mockStore({});
+    await store.dispatch(uploadImage())
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+
+  it('Should dispatch UPLOAD_IMAGE_ERROR', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 404,
+        response: {
+          data: 'error',
+        },
+      });
+    });
+    const expectedActions = [{
+      payload: {
+        data: 'error',
+      },
+      type: UPLOAD_IMAGE_ERROR,
+    }];
+    store = mockStore({});
+    await store.dispatch(uploadImage())
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+});
```

Comment:
I recommend you move mock data from test into `src/__mocks__` so that another person may use it later.